### ISSUE TITLE
Base map layers - consider changing default layer

### ISSUE THREAD

**Issue by KattyBaird** [BiologicalRecordsCentre/DECIDE-app#34]
In feedback form - standard OpenStreetMap preferred to simple map layer.
" this was quite difficult to interpret even in areas I know quite well and it was distracting not having contextual information. Also found the rights of way were not very helpful, some obvious routes missing and some erroneously put through the middle of lakes etc. The more detailed map layer was much more useful in finding access areas"

**Comment by burkmarr** [comment]
@mjopocock - are we going to change to OSM for version 0.2.0?

**Comment by mjopocock** [comment]
@burkmarr I think, given the feedback and some of the quirks with Stamen (e.g. town names disappear when zooming in), yes switch to OSM for the default. Greg is happy with that for now.
We will need to make sure that a new colour scale works well with OSM.

**Comment by burkmarr** [comment]
@mjopocock @cagatayTurkay and anyone else. Here are some screen grabs of the two new styles we agreed to try at yesterday's meeting. I've included shots over both standard OSM backdrop (which we agreed to switch to as the default) and the stamen toner light ('simple map). For each, I've included grabs at full opacity and 50%. (Don't pay any attention to the actual decide scores - I think my development setup still includes a very early version of the score maps.)

Cividis over OSM 100% opacity

![image](https://user-images.githubusercontent.com/9275178/124660517-36c4d380-de9e-11eb-84c4-04c3dd30339c.png)

Cividis over OSM 50% opacity

![image](https://user-images.githubusercontent.com/9275178/124660579-4c39fd80-de9e-11eb-837e-4fec3f5c70bd.png)

Cividis over Toner Light 100% opacity

![image](https://user-images.githubusercontent.com/9275178/124660694-74296100-de9e-11eb-8f72-5416f15602bb.png)

Cividis over Toner Light 50% opacity

![image](https://user-images.githubusercontent.com/9275178/124660658-670c7200-de9e-11eb-947b-d418cfcf28e8.png)

Truncated Viridis over OSM 100% opacity

![image](https://user-images.githubusercontent.com/9275178/124660846-ab980d80-de9e-11eb-918f-9c393e99e97b.png)

Truncated Viridis over OSM 50% opacity

![image](https://user-images.githubusercontent.com/9275178/124660891-b94d9300-de9e-11eb-8691-ea5c72e65fbd.png)

Truncated Viridis over Toner Light 100% opacity

![image](https://user-images.githubusercontent.com/9275178/124660984-d3877100-de9e-11eb-80ab-3350b39c5967.png)

Truncated Viridis over Toner Light 50% opacity

![image](https://user-images.githubusercontent.com/9275178/124660938-c7031880-de9e-11eb-9735-faba3886f902.png)

**Comment by mjopocock** [comment]
Thanks @burkmarr @GregMci @cagatayTurkay
It's a tough one, isn't it? Thanks for all the options to help us look at this!

Given the criteria that users value orientating themselves against the underlying map (OSM being preferred, for now), and that we want the high values to 'zing', **I think the best option (for now!) has to be truncated viridis with OSM background**, (although I do keep being drawn to the simplicity of the Stamen maps ;-) ). Can you tweak the opacity to 60 to help the yellow to zing while still being able to see the underlying map?

@AugustT @KattyBaird @tmondain

(It made me wonder if slightly increasing transparency of darker colours, and increasing opacity of brighter colours would help them stand out... But I suspect that will be a pain to implement and would require tweaking, which is not the best use of time right now.)

**Comment by KattyBaird** [comment]
Agree, of these I like truncated viridis on OSM and it needs an opacity less than 100%, but a comment for future versions - I do wonder if colours are too similar to the features on the underlying OSM e.g. hard distinguishing green scores from greenspace? Similar issues with some colours on cividis.  I guess an opacity slider would help overcome this...
(I personally prefer OSM base layer to the stamen one - I think because I am more familiar with interpreting it)

**Comment by TMondain** [comment]
Hi, thanks for these Rich! 

I think my preference is truncated viridis over the OSM too. I agree with Katty that it's a little hard to distinguish between green grids vs green colours on the map though. The reason I like the OSM is because I feel like it provides more information about the habitat/underlying landscape than the Stamen. I would be happy with a black and white version of the OSM, for example, but not sure if that's even possible.

Also, with the legend, I wonder if "Higher recording priority" is better because it's slightly more concise? I'm being pedantic, sorry.

**Comment by cagatayTurkay** [comment]
Thanks @burkmarr , these are looking excellent! Agree with everyone, we can go for now with the truncated Viridis over OSM.
 

As all pointed out, there will be interactions with the background map, this is only avoidable by using a muted base map such as the Stamen black/white but then the context is gone. 

In future iterations, we discussed to look into separating the layers within the background maps (separate road network, place names, rights of way, etc.) and having the DECIDE score layer sandwiched between those. This should address some of the confusion arising from the interaction between the score layer and the rest while also enabling us to show the context -- but that's a discussion for the next iteration!

**Comment by burkmarr** [comment]
> Also, with the legend, I wonder if "Higher recording priority" is better because it's slightly more concise? I'm being pedantic, sorry.

@TMondain But it's still too many words for one line, so would we go for:
```
Higer recording
       priority
```
or
```
           Higher
recoring priority
```
or something else?! Although the current one is more words, I think it looks more balanced than any of the alternative three word ones would.

**Comment by burkmarr** [comment]
Consensus seems to be for OSM (colour) with truncated viridis colour scheme so that's what I'll implement. I'll set the initial opacity to 60% in line with Michael's suggestion.

**Comment by burkmarr** [comment]
> I do wonder if colours are too similar to the features on the underlying OSM e.g. hard distinguishing green scores from greenspace? Similar issues with some colours on cividis. I guess an opacity slider would help overcome this...!

@KattyBaird - @cagatayTurkay and @GregMci have some ideas to deal with this which we will trial for the subsequent release. (There is already an opacity slider.)

**Comment by burkmarr** [comment]
(I'll leave Stamen toner light maps - called 'simple map' in legend - in the layer switcher control on the map, so still available to those that want a simple background under the decide score.)

**Comment by burkmarr** [comment]
>(It made me wonder if slightly increasing transparency of darker colours, and increasing opacity of brighter colours would help them stand out... But I suspect that will be a pain to implement and would require tweaking, which is not the best use of time right now.)

@mjopocock - the variable opacity is doable and I've demonstrated it to @cagatayTurkay and @GregMci to help with their thinking about fog maps. (The transparency is reversed in the case of the fog map - lower scores = high transparency and high scores = low transparency.) Suggest we look at this when we investigate/discuss fog maps further.

**Comment by mjopocock** [comment]
> > Also, with the legend, I wonder if "Higher recording priority" is better because it's slightly more concise? I'm being pedantic, sorry.
> 
> @TMondain But it's still too many words for one line, so would we go for:
> 
> ```
> Higer recording
>        priority
> ```
> 
> or
> 
> ```
>            Higher
> recoring priority
> ```
> 
> or something else?! Although the current one is more words, I think it looks more balanced than any of the alternative three word ones would.

I agree with sticking for the current version as in the pictures above. Thanks for thinking about this!

**Comment by TMondain** [comment]
> > Also, with the legend, I wonder if "Higher recording priority" is better because it's slightly more concise? I'm being pedantic, sorry.
> 
> @TMondain But it's still too many words for one line, so would we go for:
> 
> ```
> Higer recording
>        priority
> ```
> 
> or
> 
> ```
>            Higher
> recoring priority
> ```
> 
> or something else?! Although the current one is more words, I think it looks more balanced than any of the alternative three word ones would.

Yep, that makes sense. I agree the current one looks more balanced :)
